terraform/stacks/cdk_bootstrap/main.tf

```diff
@@ -0,0 +1,39 @@
+terraform {
+  required_version = ">= 1.5.4"
+
+  backend "s3" {
+    bucket         = "umccr-terraform-states"
+    key            = "cdk_bootstrap/terraform.tfstate"
+    region         = "ap-southeast-2"
+    dynamodb_table = "terraform-state-lock"
+  }
+
+  required_providers {
+    aws = {
+      version = "5.12.0"
+      source  = "hashicorp/aws"
+    }
+  }
+}
+
+provider "aws" {
+  region = "ap-southeast-2"
+}
+
+resource "aws_cloudformation_stack" "CDKToolkit" {
+  name = "CDKToolkit"
+  capabilities = [
+    "CAPABILITY_NAMED_IAM"
+  ]
+
+  parameters = {
+    CloudFormationExecutionPolicies = "arn:aws:iam::aws:policy/AdministratorAccess"
```

Comment:
Perhaps, we shall also start creating a dedicated IAM role in this TF stack for this role policy..? And, start tracking / updating here as need be basis - expanding CF exec permission policy as we get go..? 

I like the idea proposed in this article, as a starter.
https://betterdev.blog/cdk-bootstrap-least-deployment-privilege/

Reply:
Yes - I think there is probably a best practice approach to the IAM role and the permissions boundaries - that I guess we can work towards.
AWS seems to be doing some approach to the use of permission boundaries in the CDK bootstrap (there are extra parameters now).. but seems like a work in progress